src/test/suite/telemetry/telemetryController.test.ts

```diff
@@ -1,14 +1,11 @@
 import * as vscode from 'vscode';
-import { StorageController } from '../../../storage';
-import { TestExtensionContext } from '../stubs';
+import * as path from 'path';
```

Comment:
isn't this just `import path from 'path';`?

Reply:
I remember it was the safest way to import it. Was failing with just `import path from 'path';`, probably on windows. Don't remember exactly what.